lib/providers/services/eos_service.dart

```diff
@@ -94,8 +94,35 @@ class EosService {
     return client.pushTransaction(transaction, broadcast: true);
   }
 
+  Future<dynamic> plantSeeds({String amount}) async {
```

Comment:
This should be an integer, not a string

Reply:
fixed, also function was not used before